test/evergreen/code_coverage/code_coverage_config.json

```diff
@@ -145,6 +145,8 @@
     "test/csuite/wt3363_checkpoint_op_races/test_wt3363_checkpoint_op_races",
     "test/csuite/wt3120_filesys/test_wt3120_filesys -b $(pwd)",
     "test/csuite/wt8659_reconstruct_database_from_logs/test_wt8659_reconstruct_database_from_logs",
+    "test/csuite/incr_backup/test_incr_backup -S 123456789",
+    "test/csuite/incr_backup/test_incr_backup -S 0x9b1bde3f111fe316",
```

Comment:
It feels like maybe these cases should be also added to `test/csuite/incr_backup/smoke.sh`?

Reply:
Hi @sueloverso , that's a great suggestion, but unless I've missed something is actually not necessary. 
Here's why: With my changes and when not given a seed, incr_backup itself now runs the test three times: once each with those specific seeds, and once with a random seed. That means it will always trigger a failure without your fix to WT-10551 (because one of those seeds generates a failure). The reason I'm running with a seed (twice) in the code coverage is to specifically avoid the non-deterministic random seed test.
This means that the default (as executed by smoke.sh) will run with those two seeds and a random one already.